.travis.yml

```diff
@@ -50,6 +50,35 @@ matrix:
             - git
             - build-essential
       env: SUITE=ci-all-features
+    - rust: 1.33.0
+      sudo: true
+      before_install:
+        - sudo apt-get update
+      addons:
+        apt:
+          packages:
+            - git
+            - build-essential
+            - libcurl4-openssl-dev
+            - libelf-dev
+            - libdw-dev
+            - cmake
+            - gcc
+            - binutils-dev
+      script:
+        wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz &&
```

Comment:
Can we lock a specific release? In my experience tools working on binary levels are usually ... flaky, and it might resolve in unwanted error in a total different PR if upstream is updated somehow.

Reply:
We can download zip from [releases](https://github.com/SimonKagstrom/kcov/releases)

Reply:
updated